keep-alive/src/main/java/KeepAliveGatt.kt

```diff
@@ -72,25 +71,45 @@ sealed class State {
     override fun toString(): String = javaClass.simpleName
 }
 
+sealed class Event {
+    data class Connected(val gatt: GattIo) : Event()
+    data class Disconnected(val disconnectInfo: DisconnectInfo) : Event()
+}
+
+data class DisconnectInfo(
+    val wasConnected: Boolean,
+    val connectionAttempt: Int
```

Comment:
Should this be `connectionAttempts` to imply past tense?
Or maybe `connectionsAttempted` would be even clearer?
Maybe even `connectionAttemptsSinceConnected`? Too verbose?

Reply:
I wasn't sure about this one. I feel like this data object represents a collection of information about a specific connection attempt so 'connectionAttempt' describes which # attempt it was.

I think 'wasConnected' could stand to be renamed, but I can't think of anything better here. Any ideas?

Considering I am no longer resetting the number after connecting, I think connectionAttemptsSinceConnected would no longer make sense here.